Update/omot_omake.txt

```diff
@@ -3402,7 +3402,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 5, "ps3/s20/05/440500208", 256, TRUE);
 	OutputLine(NULL, "「でも、与えられるよりも与える立場になって、誰かの時間と世界をあたたかなものに変えることは、きっとそれ以上に素敵だと思うのですよ」",
-			NULL, "", Line_Normal);
+			NULL, "\"But I believe that rather than being given, it is more wonderful to be in the position of giving to change someone's time and world into something blissful.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, "\"But I believe that rather than being on the receiving end, it is more wonderful to be the one giving, to change someone else's time and world into something blissful.\"", Line_Normal);
```